set.mm

```diff
@@ -659489,12 +659495,13 @@ D Fn ( I ... ( M - 1 ) ) /\
     finsubmsubg.g $e |- ( ph -> G e. Grp ) $.
     finsubmsubg.s $e |- ( ph -> S e. ( SubMnd ` G ) ) $.
     finsubmsubg.1 $e |- ( ph -> B e. Fin ) $.
-    $( A submonoid of a finite group is a subgroup.  This does not extend to
-       infinite groups, even if we include the inverses of the elements of the
-       submonoid; for example, ((the submonoid ` S = A + B sqrt 2 ` where
-       ` A , B e. NN ` ) together with their negations) only contains elements
-       with absolute value greater than 1, so it is not closed under
-       subtraction: ` -u 1 + sqrt 2 e/ S ` .  (Contributed by SN,
+    $( A submonoid of a finite group is a subgroup.  However, this does not
+       extend to infinite groups, even when including the inverses of the
+       elements of the submonoid.  For example, taking the submonoid (of the
+       group of real numbers under addition) ` S = A + B sqrt 2 ` where
+       ` A , B e. NN0 ` (together with their negations), ` S ` only contains
+       zero and elements whose absolute value is at least 1, so it is not
+       closed under subtraction: ` sqrt 2 - 1 e/ S ` .  (Contributed by SN,
        31-Jan-2025.) $)
```

Comment:
I think you're doing two steps at once, which may make the comment harder to understand.  I would write something like:
```
A submonoid of a finite group is a subgroup.  This does not extend to infinite groups, as the
submonoid ` NN0 ` of the group ` ( ZZ , + ) ` shows.  Note also that the union of a submonoid
and its opposite need not be a submonoid, as the submonoid ` ( NN0 \ { 1 } ) ` of the group
` ( ZZ , + ) ` shows: 3 is in that submonoid, -2 is in its opposite, but 1 is not in their union.
Actually, the subgroup generated by ` ( NN0 \ { 1 } ) ` is ` ZZ ` .
```